modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,91 @@
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd #<1>
+spec:
+  controller:
+    respectRBAC: strict #<2>
+----
+<1> Specify the name of the Argo CD instance.
+<2> Specify the value of the `.spec.controller.respectRBAC` key in the Argo CD resource. You can set it to `normal` or `strict`. If the value is set to `strict`, Argo CD performs additional validations of RBAC resources to determine permissions. This setting increases the number of API calls to the server as it is more accurate compared to `normal`. Switch to `strict` if Argo CD reports errors indicating it cannot access resources when the value is set to `normal`. If the value is set to `normal`, Argo CD determines its RBAC permissions by attempting to list each resource. By default it is recommended to start with `normal`, as it balances accuracy and speed because resource listing is a lightweight operation.
+
+. Apply the changes to the YAML object file by running the following command.
++
+[source,terminal]
+----
+$ oc apply -f argocd-resource.yaml -n argo-cd-instance #<1>
+----
+<1> Specify the name of the YAML file that includes the Argo CD resource and the namespace that hosts the Argo CD instance.
++
+. Verify the successful installation of Argo CD by checking the status of the ArgoCD `.status.phase` field. The {gitops-shortname} Operator sets this field to `Available` when the installation is complete. Run the following command to check the status:
++
+[source,terminal]
+----
+kubectl get argocd <argocd-instance-name> -o jsonpath='{.status.phase}' #<1>
+----
+<1> Replace `<argocd-instance-name>` with the name of your Argo CD instance for example, `example-argocd`.
+
+[id="configuring-respectRBAC-using-the-web-UI_{context}"]
+== Configuring respectRBAC using the Web UI
+
+You can configure `respectRBAC` in the Web UI.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Configure the `respectRBAC` parameter in the *Argo CD* tab.
++
+[source,yaml]
+----
+spec:
+  controller:
+    respectRBAC: strict
+----
+
+. Click *Create*.
++
+After successful installation, the Argo CD instance is listed under *Argo CD* tab and its Status should be set to *Available*.
+
+. After the Argo CD instance is created, verify that the `resource.respectRBAC` parameter in the config map resource is updated successfully by completing the following steps.
+
+.. In the *Administrator* perspective, go to *Workload* -> *ConfigMaps*.
+.. In the *Project* option, select *Argo CD* namespace.
+.. Select the `argocd-cm` config map.
+.. Select the *YAML* tab to view the `resource.respectRBAC` parameter.
```

Comment:
This section is missing in the CLI procedure part

Reply:
To ensure that I understand this feedback correctly, do you want me to add these steps in the CLI section of the document **as-is** after the 3rd step?

Reply:
As is wouldn't be a pure CLI way of doing it so I would recommend to add something like

`oc get cm argocd-cm -n <argocd namespace> -o yaml` for verifying the content

Reply:
Ok, this command would be used to that the `resource.respectRBAC` parameter in the config map resource is updated successfully? And it would be the 4th step in the CLI section?